packages/components/pie-cookie-banner/test/helpers/page-object/pie-cookie-banner.page.ts

```diff
@@ -293,4 +293,20 @@ export class CookieBannerComponent extends BasePage {
         }));
         return Promise.all(elements);
     }
+
+    async setProperty (property: string, value: string) {
+        await this.page.evaluate(([prop, val]) => {
+            const component = document.querySelector('pie-cookie-banner');
+            if (component) {
+                component[prop] = val;
+            }
+        }, [property, value]);
+    }
```

Comment:
This is great! @JoshuaNg2332 I believe its related to what I was looking for in Modal